### ISSUE TITLE
network inspect does not have format

### ISSUE THREAD

**Issue by medyagh** [containers/libpod#5205]
we added podman as a driver to minikube ! and merged but I wanna add minikube's mount feature. https://github.com/kubernetes/minikube/issues/6628

I wanna do something simmilar to this in docker:
```
docker inspect --format '{{(index .IPAM.Config 0).Gateway}}' $(docker network ls --filter name=bridge --format {{.ID}})
```

but podman doesnt have format for network inspect

```
 podman network inspect --help
network inspect
Description:
  Inspect network
Usage:
  podman network inspect NETWORK [NETWORK...] [flags] 
Examples:
  podman network inspect podman
```

**Comment by vrothberg** [comment]
Thanks for opening the issue, @medyagh! Are you interested in contributing this feature?

**Comment by baude** [comment]
@medyagh the network stack is a major difference between podman and docker.  for example, in inspect there really is very little chance the output would look the same.  given that, would format still be helpful?

**Comment by medyagh** [comment]
@baude all i need is a way to get the routable IP of the host from inside of the container.

so if I have a webserver running on the host machine, 
from inside the container, I wanna be able to ping that IP . 

in docker I get that IP using this command:
```
docker inspect --format '{{(index .IPAM.Config 0).Gateway}}' $(docker network ls --filter name=bridge --format {{.ID}})
```
i want same command for podman (so I can implement minikube mount command)

**Comment by rhatdan** [comment]
This is definitely something we want.  Just need someone to take it up and run with it.
@kunalkushwaha @sujil02 Anyone interested?

**Comment by kunalkushwaha** [comment]
I can pick this.

/assign

**Comment by rhatdan** [comment]
@kunalkushwaha What is going on with this issue?

**Comment by kunalkushwaha** [comment]
Sorry, somehow it skipped from task list. Its half done.. shall make PR soon.

**Comment by medyagh** [comment]
@kunalkushwaha are there anything new about this ?

**Comment by kunalkushwaha** [comment]
@medyagh Sorry for the delay in this. Due to recent client code (v2 changes), my fix is not mergeable. 
I am trying to fix this. Shall be able to do it soon.

**Comment by afbjorklund** [comment]
I think it would be better to look at the container's gateway ? Or am I missing something here.

**Comment by QiWang19** [comment]
This issue #5205 was fixed by #6161
